daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/value/ValueCoder.scala

```diff
@@ -515,9 +521,19 @@ object ValueCoder {
             }
             builder.setMap(protoMap).build()
 
-          case ValueGenMap(_) =>
-            // FIXME https://github.com/digital-asset/daml/issues/2256
-            throw Err("Trying to serialize GenMap which are not currently not supported.")
+          case ValueGenMap(entries) =>
+            val protoMap = proto.GenMap.newBuilder()
+            entries.foreach {
+              case (key, value) =>
+                protoMap.addEntries(
+                  proto.GenMap.Entry
+                    .newBuilder()
+                    .setKey(go(nesting, key))
```

Comment:
Does this need `newNesting` as well here?

Reply:
fixed. Thanks